src/tests/internals.helpers.test.js

```diff
@@ -173,3 +173,21 @@ describe('pickXKey()', () => {
     expect(xKey).toBeUndefined();
   });
 });
+
+describe('processNode()', () => {
```

Comment:
im not a big fan of testing the internals, i dont like this file in general.

I think what could be better would be two schemas, one like you have and then a `true` case in the `conditions.test.js` file, just testing it from the top level API perspective

Other than that, this change works great

Reply:
Thanks for the review @brennj. It makes total sense! I updated the code. Please, let me know what you think 🙂